package.json

```diff
@@ -19,21 +19,22 @@
   "author": "Leonardo Andres Garcia Crespo <leoasis@gmail.com>",
   "license": "ISC",
   "homepage": "https://github.com/leoasis/react-sound",
-  "peerDependencies": {
-    "soundmanager2": "^2.97.20150601-a",
-    "react": ">=0.13.3 <16.0.0"
+  "dependencies": {
+    "react": "^15.1.0",
```

Comment:
Done, I kept `react-dom` as a devDependency just to be able for us to modernize the examples.